html/js/product-history.js

```diff
@@ -0,0 +1,54 @@
+// This file is part of Product Opener.
+//
+// Product Opener
+// Copyright (C) 2011-2024 Association Open Food Facts
+// Contact: contact@openfoodfacts.org
+// Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+//
+// Product Opener is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+/*global revert_confirm_message*/
+/*exported activate_product_revert_buttons_in_history*/
+
+function activate_product_revert_buttons_in_history () {
+    $('#history_list a.button').on('click', function() {
+        const code = $(this).data('code');
+        const rev = $(this).data('rev');
+        // using confirm, could be replaced with some JS dialog / modal
+        const confirm = window.confirm(revert_confirm_message); // eslint-disable-line no-alert
+        if (confirm) {
+            $.ajax({
+                url: '/api/v3/product_revert',
+                type: 'POST',
+                contentType: "application/json; charset=utf-8",
+                dataType: "json",
+                data: JSON.stringify({
+                    code: code,
+                    rev: rev,
+                    fields: "rev"
+                }),
```

Comment:
Shan't there be the lc and cc parameters ? (so that we get back localized product name ?)

Reply:
The localized status / error name you mean? (we don't display the product name)
It will get the right default value from the subdomain:

![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/8158668/2a84e312-e4cb-4632-a43d-30195e18661d)
(tested with a translation added manually in fr.po)

Reply:
ok !
Maybe add a comment to explain it ? (for the case we evolve it later) ?

Reply:
Sure